code/datums/spell.dm

```diff
@@ -461,13 +461,30 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell))
 			target.vars[type] += amount //I bear no responsibility for the runtimes that'll happen if you try to adjust non-numeric or even non-existant vars
 	return
 
-/obj/effect/proc_holder/spell/aoe_turf
+/obj/effect/proc_holder/spell/aoe
 	create_attack_logs = FALSE
 	create_custom_logs = TRUE
+	/// How far does it effect
+	var/aoe_range = 7
+
+/obj/effect/proc_holder/spell/aoe/create_new_targeting() // Fallback to turf if someone is too moronic to use a proper subtype
+	var/datum/spell_targeting/aoe/turf/targeting = new()
+	targeting.range = aoe_range
+	return targeting
```

Comment:
dont do this. we have a CI check to check for this

Reply:
yes and if I don't do this CI fails